main_f4.c

```diff
@@ -411,6 +412,8 @@ main(void)
 	rcc_clock_setup_hse_3v3(&clock_setup);
 
 	/* start the interface */
+	char * sn = "SOME_SERIAL_NO";
```

Comment:
This will have to be changed to read in the UDID of the chip. I currently have the problem that I can't convert the UDID to a string because I don't have "sprintf" compiled it. 

Anyone have any ideas how I can elegantly get the UDID serial into a string?